### PR TITLE
It6 erb

### PR DESCRIPTION
**Pull request by nancylee713**
Attempt to make a website
- Embed ruby codes into an html with special syntax `<%  %> and <%=  %>`
- Use the following command on terminal to convert the `.html.erb` file to `.html`.
```ruby
$ erb site/index.html.erb > site/index.html
```
- Add styling using bootstrap and css

Also attempt to incorporate mocks/stubs in one of the test methods (`team_info`) but it might not make sense to stub a method when we're trying to test its functionality.

### CODE REVIEW

File: test/team_statistics_test.rb [hunk 1]
```diff
@@ -19,6 +12,9 @@ def test_team_info
       "abbreviation" => "BOS",
       "link" => "/api/v1/teams/6"
     }
+
+    stat_tracker = stub(team_info: expected)
```
Comment (StarPerfect) [inline test/team_statistics_test.rb:16]: .returns({'key'=>'value', 'key2' => 'value2', etc}) this is the format Ian provided me to do a hash as a stub so we would write something like this:
stat_tracker = stubs(:hash_method_name).returns({'key'=>'value', 'key2' => 'value2', etc})

**Comment by fentontaylor** [comment]
This is really cool Nancy. Great job! I can't wait to play around with it some more.

File: test/team_statistics_test.rb (hunk 1 above)
Comment (nancylee713) [reply test/team_statistics_test.rb:16]: I see. I'll modify the `team_info` method and update this PR shortly. Thanks Corina!

Comment (nancylee713) [reply test/team_statistics_test.rb:16]:
```ruby
  def test_team_info
    expected = {
      "team_id" => "6",
      "franchise_id" => "6",
      "short_name" => "Boston",
      "team_name" => "Bruins",
      "abbreviation" => "BOS",
      "link" => "/api/v1/teams/6"
    }

    stat_tracker = stubs(:team_info).with(6).returns(expected)

    assert_equal expected, stat_tracker.team_info("6")
  end
```

When I run the above code, it throws an error message:

```
Error:
TeamStatisticsTest#test_team_info:
NameError: undefined local variable or method `team_info' for #<TeamStatisticsTest:0x00007f8c848a0af8>
Did you mean?  test_team_info
    test/team_statistics_test.rb:16:in `test_team_info'
```

Any idea why the test is failing?

Comment (fentontaylor) [reply test/team_statistics_test.rb:16]: looks like you used an integer in one place, but a string in the other. Try this:
`stat_tracker = stubs(:team_info).with('6').returns(expected)` with the 6 as '6' instead.

Comment (StarPerfect) [reply test/team_statistics_test.rb:16]: Sorry guys maybe i wasn't clear with my first message. From what I understood from Ian it should look like this:
 stat_tracker = stubs(:team_info).returns({'team_id'=>'6', 'franchise_id' => '6', 'short-name' => 'Boston', 'team_name' => 'Bruins', 'abbrevation' => 'BOS', 'link' => '/api/v1/teams/6'})

Comment (StarPerfect) [reply test/team_statistics_test.rb:16]: Let me know if that works or what happens after running that Nancy. Great job getting started on the Iteration 6 by the way!!!

Comment (nancylee713) [reply test/team_statistics_test.rb:16]: Still no pass :(

**Pull request closed without merging**